datalad/distribution/install.py

```diff
@@ -285,8 +282,8 @@ class Install(Interface):
     @staticmethod
     @datasetmethod(name='install')
     def __call__(
+            source,
             path=None,
-            source=None,
```

Comment:
I will not comment in general yet. First, I want to see it done in full and how it feels then. But just re this point:

`datalad install ds*` -> `datalad get ds*`